website/src/docs/hotchocolate/defining-a-schema/subscriptions.md

```diff
@@ -256,6 +256,8 @@ public async Book PublishBook(Book book, [Service] ITopicEventSender sender)
 
 If more complex topics are required, we can use the `ITopicEventReceiver`.
 
+> ⚠️ Note: Pay attention to the use of `SubscribeAndResolve` instead of `Subscribe`. As can be seen in the code-first example above, all subscription fields are in fact made of two methods (resolver and subscriber). When using `Subscribe`, the attributed method represents resolver method (with subscriber method generated at runtime) whereas `SubscribeAndResolve` denotes subscriber method (with resolver method generated at runtime).
```

Comment:
For the SubscribeAndResolve part I think it would be more understandable, if you write something like "each event message is directly returned as the subscription resolver result".

Reply:
What do you mean? Each iteration of the IAsyncEnumerable is followed by incovation of the resolver, in this case the "shallow" resolver looks like ctx => ctx.GetMsg<Foo>() so it just directly returns the message.

It could mention the shape of the compiled delegate, but I do not think it wise to skip the part where this method's body is in fact the subscriber method (as seen in SubscribeAndResolveAttribute's code). It also explains why UseProjectionAttribute won't work here (as it is not a resolver method).

Reply:
@tobias-tengler